content/en/docs/reference/glossary/endpoint.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Endpoint
+id: endpoint
+date: 2020-04-23
+full_link: 
+short_description: >
+  Endpoint tracks the IP Addresses of the objects the service send traffic to.
+
+aka:
+tags:
+- fundamental
+---
+ Endpoints tracks the IP Addresses of the objects the service send traffic to.
+
+<!--more-->
+
+Endpoint can be updated automatically through labels and selectors, or you can manually configure your endpoints depending on your use case.
```

Comment:
If you want to, you can also make “labels” and “selectors” be glossary tooltips.